sklearn/ensemble/tests/test_bagging.py

```diff
@@ -77,7 +79,32 @@ def test_classification():
                               **params).fit(X_train, y_train).predict(X_test)
 
 
-def test_sparse_classification():
+@pytest.mark.parametrize(
```

Comment:
An alternative to `product` is:

```py
@pytest.mark.parametrize("sparse_format", [csc_matrix, csr_matrix])
@pytest.mark.parametrize("params, method", [...])
def test_sparse_classification(sparse_format, params, method):
	...
```

Reply:
Got it. But I'm not quite sure which way is better. @rth @thomasjpfan Are these two styles are both acceptable?

Reply:
Either one one is fine with a slight preference for separate `parametrize`, I think. No need to change.